datahub-web/package.json

```diff
@@ -71,5 +71,9 @@
   },
   "dependencies": {
     "yarn": "^1.10.1"
```

Comment:
Do we still need this line?

Reply:
I think we do. Volta is its own special tooling environment that reads from the "volta" property. However, in this case it does not interact with any other part of the client build logic which should still rely on this dependencies line if checking what yarn version we are using. Though the two versions should match haha.